pkgs/development/python-modules/svgelements/default.nix

```diff
@@ -0,0 +1,35 @@
+# https://github.com/meerk40t/svgelements
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest
+, numpy
+, scipy
+}:
+
+buildPythonPackage rec {
+  version = "1.6.12"; # not quite!
```

Comment:
Generally, when not using the upstream release we tag it `unstable-YEAR-MONTH-DAY`. This however is not a established convention. Some use `VERSION-REF`, which in this case is `1.6.12-761bb3`.